pantheon/src/main/java/tech/pegasys/pantheon/cli/PantheonCommand.java

```diff
@@ -238,19 +239,19 @@ void setBootnodes(final List<String> values) {
   private final Integer maxPeers = DEFAULT_MAX_PEERS;
 
   @Option(
-      names = {"--limit-remote-wire-connections-enabled"},
+      names = {"--remote-connections-limit-enabled"},
```

Comment:
Question - why did we end up with 2 options here (enabled + percentage), instead of 1 (just percentage)?  If the percentage is at 1, the limit is effectively disabled.

Reply:
We agreed that this is such an important option that the user should explicitly enable this protection. It has a strong impact on the behaviour of the node.

Reply:
It is easier from a UX perspective to configure with a feature flag and a value flag.  I also ran this by Jake and there are many places we do this so it is a well worn pattern now.